### DISCUSSION TITLE
Siri Shortcuts and 'Workflow ID'

### DISCUSSION THREAD

**Discussion by ShonFrazier** [community/community#11789] [Mobile]
In the Shortcuts app, I can see that GitHub Mobile offers a 'Dispatch Workflow' shortcut. Setting this up requires a 'Workflow ID' but I can't work out how to discover the ID of the actions in my repository. Can someone describe where these IDs can be found, or point me to documentation that explains?

**Comment by eliperkins** [comment]
Hey @ShonFrazier! Thanks for this question about the Dispatch Workflow shortcut.

The Workflow ID should be the name of the YAML file that defines your workflow. So if you had a file in your repository at `.github/workflows/main.yaml`, the Workflow ID would be `main.yaml`. There's a little bit more documentation on this within the [REST API docs for "Create a Workflow Dispatch Event" here](https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event).

Let me know what other questions you have about using this shortcut!

**Comment by vetqa0-gorzup-sYqwuv** [comment]
How can I get a bitcoin wallet hacker
